src-electron/zcl/zcl-loader-silabs.js

```diff
@@ -1526,15 +1526,29 @@ async function processBitmapFields(
     data.features.length == 1 &&
     'feature' in data.features[0]
   ) {
-    data.features[0].feature.forEach((item) => {
-      bitmapFields.push({
-        bitmapName: 'Feature',
-        bitmapClusterCode: [{ $: { code: data.code[0] } }],
-        name: item.$.name,
-        mask: 1 << item.$.bit,
-        fieldIdentifier: item.$.bit
+    let clusterCodes = [{ $: { code: data.code[0] } }]
+    if ('cluster' in data.features[0] && data.features[0].cluster.length > 0) {
+      data.features[0].cluster.forEach((clCode) => {
+        if (
+          !clusterCodes.some(
+            (existingCode) => existingCode.$.code === clCode.$.code
+          )
+        ) {
+          clusterCodes.push({ $: { code: clCode.$.code } })
+        }
       })
-    })
+    }
+    clusterCodes.forEach((clCode) =>
+      data.features[0].feature.forEach((item) => {
+        bitmapFields.push({
+          bitmapName: 'Feature',
+          bitmapClusterCode: [clCode],
+          name: item.$.name,
+          mask: 1 << item.$.bit,
```

Comment:
I wonder if this blows up with a very weird error if "bit" in XML is not a nice number.... Might be better to extract "bit" out first, and if it's not a number, throw some clearly specified error, otherwise people might be confused.